### PR TITLE
Implements the wage gap

### PR DESCRIPTION
**Pull request by LordFowl**
Modifies the money available in employee accounts in the following way based on gender;
Females make 80% of the normal wage they would have received
Neuters or genderless make 70% of the normal wage they would have received.

This applies in addition to RNG, species mods, loyalty mods, and job mods.

Marginalized races also are less likely to be selected for certain jobs, along with women.

Feedback thread; https://forums.aurorastation.org/topic/11486-wage-gap/

### CODE REVIEW

**Comment by LordFowl** [comment]
!review

File: code/game/jobs/job/job.dm [hunk 1]
```diff
@@ -84,7 +84,18 @@
 		species_modifier = human_species.economic_modifier
 		PROCLOG_WEIRD("species [H.species || "NULL"] did not have a set economic_modifier!")
 
-	var/money_amount = (rand(5,50) + rand(5, 50)) * loyalty * economic_modifier * species_modifier
+	var/wage_gap
+	switch(H.gender)
+		if(MALE)
```
Comment (skull132) [inline code/game/jobs/job/job.dm]: Duplicated under the default case. Remove this case.

**Comment by Conspiir** [comment]
It's 2461. We've had contact with multiple alien species. What's the justification for gender to still be a factor in wage? Particularly since the oldest-contacted species, Skrell, make no distinction in gender and would not suffer differing payments for equal work?

**Comment by Supermichael777** [comment]
Additionally 70% is based on an inaccurate way of representing data (average salary of women/average for men).  In same group pay the disparity is less than 2%, quite often its nothing.  Primarily it reflects the low number of women making extremely large salaries, and even that's changing to the point where the "gap" may cease to exist within our lifetimes.  Its just that executive salaries outweigh normal people by 30000% 

So if you want to be realistic just make all NT executives men.

**Comment by Raze500** [comment]
Better, ¡Womans get 95% And the disgusting males gets 20% And the genderless gets 80%!

**Comment by AffectedArc07** [comment]
Me likely

**Comment by AldenBluetenberger** [comment]
> Better, ¡Womans get 95% And the disgusting males gets 20% And the genderless gets 80%!

no

**Comment by mattroks101** [comment]
I can't even tell if this is a meme PR or not.

**Comment by Arrow768** [comment]
Limit the discussion on github to technical feedback.

**Comment by skull132** [comment]
Both maintainers have deemed the idea a bad one. So, this is denied for the time being. Reasons can be found in the feedback thread.

**Pull request closed without merging**